stable/kubernetes-dashboard/values.yaml

```diff
@@ -0,0 +1,22 @@
+# Default values for kubernetes-dashboard
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+image: gcr.io/google_containers/kubernetes-dashboard-amd64
+imageTag: "v1.6.0"
+imagePullPolicy: "IfNotPresent"
+
+namespace: kube-system
+
```

Comment:
The intention is for it to work just like all other kube-system namespeced things by default. Yeah, its a little wonky though.... There's going to be a bunch of other packages like this too. Should we talk more about it?